cv_generator/main/models.py

```diff
@@ -33,4 +33,16 @@ class PersonalInformation(models.Model):
     city = models.CharField(max_length=50, blank=False)
 
     def __str__(self):
-        return f'{self.name} | {self.email}'
\ No newline at end of file
+        return f'{self.name} | {self.email}'
+    
+class AcademicExperience(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    institution = models.CharField(max_length=255, blank=False)
+    course = models.CharField(max_length=255, blank=False)
+    degree = models.CharField(max_length=255, blank=False)
+    summary = models.TextField(blank=False)
+    location = models.CharField(max_length=255, blank=False)
+    date_achieved = models.DateField(blank=False)
```

Comment:
@amikable-tech looks great 😃. Just a  few minor things that came to mind:

1. Some degrees come with a score and we should add the ability to show the score.  

Could look like this:

```python
score = models.FloatField(blank=True)
```
2. The `date_achieved` field only gives us an indication when it was achieved but not the duration. I would suggest to rather user start and end DateFields. And make the end field `blank=True` to indicate that its in progress.

Reply:
Thank you!!
will implement your suggestions now. 

> @amikable-tech looks great 😃. Just a few minor things that came to mind:
> 
>     1. Some degrees come with a score and we should add the ability to show the score.
> 
> 
> Could look like this:
> 
> ```python
> score = models.FloatField(blank=True)
> ```
> 
>     2. The `date_achieved` field only gives us an indication when it was achieved but not the duration. I would suggest to rather user start and end DateFields. And make the end field `blank=True` to indicate that its in progress.